docs/Changelog.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
```

Comment:
nit: we should use `Change Log` to be uniform with our app, and `Change_Log.md` for the file for consistency

I prefer changelog, but it's not a word